apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2.java

```diff
@@ -27,50 +28,52 @@
 import org.hl7.fhir.r4.model.Patient;
 import org.hl7.fhir.r4.model.Period;
 import org.hl7.fhir.r4.model.StringType;
+import org.springframework.stereotype.Component;
 
 /** Transforms CCW {@link Beneficiary} instances into FHIR {@link Patient} resources. */
-final class BeneficiaryTransformerV2 {
+@Component
+public class BeneficiaryTransformerV2 {
+
+  /** The Metric registry. */
+  private final MetricRegistry metricRegistry;
+
+  /** Enumerates the options for the currency of an {@link Identifier}. */
+  public enum CurrencyIdentifier {
+    /** Represents a current identifier. */
+    CURRENT,
+    /** Represents a historic identifier. */
+    HISTORIC;
+  }
```

Comment:
I generally put inner classes and enums at the bottom (though I may violate this from time to time...).  Curious why this was moved from the bottom to the top?

Reply:
Combo of style i've seen in the past (and in other classes) and personal preference; not sure what best practice is for this. I'd generally seen enums declared at the top since they are generally not huge constructs (like inner classes may be) and if they're within a class they are probably important to the understanding of that class' functionality. I personally like all vars/enums declared at the top so reading top-to-bottom you have a good idea of all the class level declarations first.